src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c

```diff
@@ -420,15 +508,13 @@ ert_release_slot(struct xocl_ert_user *ert_user, struct ert_user_command *ecmd)
 static void
 ert_post_process(struct xocl_ert_user *ert_user, struct ert_user_command *ecmd)
 {
-	struct ert_mb_validate_cmd cmd = {0}; 
-
-	if (unlikely(ert_special_cmd(ecmd)))
+	if (likely(!ert_special_cmd(ecmd)))
```

Comment:
In ert_post_process(), you copy some info from CQ slot 0 to host. But it is called in process_ert_cq(). My concern is this logic required that process_ert_sq() doesn't reuse slot 0 for other command. Right?

Reply:
post_process() called before we release the slot, at that moment the slot 0 is marked as busy. 

So no one can use slot 0 before this command processed and slot been released. 

And we only have one thread to process ert_process_cq and ert_process_rq, so before we have done processing all the cmd in cq, no body can cut the line to submit another slot 0 command

Reply:
I see. Sounds good.